apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.csproj

```diff
@@ -25,6 +25,7 @@
   <ItemGroup>
     <PackageReference Include="ConfigureAwaitChecker.Analyzer" Version="1.0.0" PrivateAssets="All" />
     <PackageReference Include="Google.Api.Gax.Rest" Version="2.2.1" />
+    <PackageReference Include="Google.Apis" Version="1.31.0" />
     <PackageReference Include="Google.Apis.Storage.v1" Version="1.30.0.1014" />
```

Comment:
Update Google.Apis.Storage.v1 to 1.31 ?

Reply:
That didn't exist when I started :) Will do, and remove this explicit one.